app/Http/Requests/CharacterUpdateRequest.php

```diff
@@ -19,7 +19,7 @@ public function rules()
             'first_name' => ['string'],
             'last_name' => ['string'],
             'date_of_birth' => ['string'],
-            'gender' => [Rule::in(['male', 'female'])],
+            'gender' => [Rule::in([0, 1])],
```

Comment:
I didn't make it an integer cause someone could then submit values that are not 1 or 0